src/components/Table/index.js

```diff
@@ -24,7 +45,9 @@ export class CustomTable extends React.Component {
                     )}
                     <StyleTbody>
                         {data &&
-                            data.map((rowData, rowIndex) => (
+                            data.length &&
+                            dataToShow &&
```

Comment:
hm?
![image](https://user-images.githubusercontent.com/32713470/57315641-4a285880-70f4-11e9-8fd1-41530def2fd4.png)

Reply:
@ejulcsi it's good to try it w another empty account in metamask to check this kind of things